lib/elephant_in_the_room_web/controllers/category_controller.ex

```diff
@@ -1,19 +1,18 @@
 defmodule ElephantInTheRoomWeb.CategoryController do
   use ElephantInTheRoomWeb, :controller
 
-  alias ElephantInTheRoom.{Sites, Repo}
-  alias ElephantInTheRoom.Sites.{Category, Site}
-  plug(:assign_site)
+  alias ElephantInTheRoom.Sites
+  alias ElephantInTheRoom.Sites.Category
 
-  def action(conn, _) do
+  def action(conn, _params) do
     site = Sites.get_site!(conn.params["site_id"])
     args = [conn, conn.params, site]
     apply(__MODULE__, action_name(conn), args)
   end
```

Comment:
Is possible to create a `pug` and add to the `pipes`, so that `conn` can have the field `site`? In this way any other functionality would have it, without modifying the default action calling mechanism in every controller who need to know in which site is working.